scripts/globals/spells/summoning/carbuncle.lua

```diff
@@ -21,6 +22,14 @@ end
 
 spell_object.onSpellCast = function(caster, target, spell)
     xi.pet.spawnPet(caster, xi.pet.id.CARBUNCLE)
+
+    if caster:hasStatusEffect(xi.effect.AVATARS_FAVOR) then
+        local effect = caster:getStatusEffect(xi.effect.AVATARS_FAVOR)
+        effect:setPower(1) -- resummon resets effect
```

Comment:
Does this if condition ever trip? Does setting power mean you could infinitely gain favor buffs since the original not deleted? This comment applies to all avatars. It would probably be safer to just delete avatar's favor on summon instead of hack around it. Going to go check on retail how this works because I'm genuinely curious

Reply:
Ah I see how it works. There's no problem here. Just staring at the code didn't get me the whole picture. Avatar's favor is the buff that activates the actual favor aura on your avatar, and you're just using it to track the tick strength for the favor.

You're good!